src/main/java/se/sundsvall/casedata/service/scheduler/supensions/SuspensionWorker.java

```diff
@@ -0,0 +1,39 @@
+package se.sundsvall.casedata.service.scheduler.supensions;
+
+
+import static java.time.OffsetDateTime.now;
+
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import se.sundsvall.casedata.api.model.PatchErrand;
+import se.sundsvall.casedata.api.model.Suspension;
+import se.sundsvall.casedata.integration.db.ErrandRepository;
+import se.sundsvall.casedata.service.ErrandService;
+
+
+@Component
+public class SuspensionWorker {
+
+	private final ErrandRepository errandRepository;
+
+	private final ErrandService errandService;
+
+	public SuspensionWorker(final ErrandRepository errandsRepository, final ErrandService errandService) {
+		this.errandRepository = errandsRepository;
+		this.errandService = errandService;
+	}
+
+	@Transactional
```

Comment:
Will this mean that all records are rolled back, despite being successful, if the last one fails?
Maybe put the `@Transactional` on `errandService.updateErrand()` instead?

Thoughts?

Reply:
Good idea